src/constants.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+
+USER = 'polkadot'
+HOME_PATH = Path('/home/polkadot')
+BINARY_PATH = Path(HOME_PATH, 'polkadot')
+CHAIN_SPEC_PATH = Path(HOME_PATH, 'spec')
+NODE_KEY_PATH = Path(HOME_PATH, 'node-key')
+DB_CHAIN_PATH = Path(HOME_PATH, '.local/share/polkadot/chains')
+DB_RELAY_PATH = Path(HOME_PATH, '.local/share/polkadot/polkadot')
+WASM_PATH = Path(HOME_PATH, 'wasm')
```

Comment:
:tada: 

Big fan of const vars separated like this, it's nice :+1:

Reply:
Not necessarily for this PR.

Just thought of something: it's not super clear what is and what isn't a directory with these. E.g. `NODE_KEY_PATH` is the path to a file, while `WASM_PATH` is the path to a directory. Maybe we should do some clearer naming?

Suggestion: `_DIR` for directories and `_FILE` for files. The current `_PATH` is already implicit since the type of the variable is `Path`.